compiler/ir/ir.tree/src/org/jetbrains/kotlin/ir/util/AdditionalIrUtils.kt

```diff
@@ -85,11 +85,21 @@ fun IrSimpleFunction.overrides(other: IrSimpleFunction): Boolean {
 private val IrConstructorCall.annotationClass
     get() = this.symbol.owner.constructedClass
 
+val IrClass.packageFqName: FqName?
```

Comment:
Looks like this is now unused

Reply:
It's needed for part in K/N, I'm now in process with these changes. I'll a bit change this method trying to use firstly data from signature if it's possible to escape extra iterations.